### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#95] [General]
1. 重写用户界面
2. 支持英语（感谢 @neoKushan）和俄语（部分，感谢 @ButtERRbrod）
3. 支持开机启动，支持自动恢复全屏

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by nt-cubic** [comment]
打开缩放配置会闪退，这是正常状况吗？

**Reply by Blinue** [reply]
麻烦发给我日志（logs\Magpie.log）

**Comment by nt-cubic** [comment]


**Reply by Blinue** [reply]
这个按钮的作用是打开同级目录下的 ScaleModels.json，出于某种原因失败了（比如被杀毒软件阻止）

**Reply by Blinue** [reply]
[Magpie.zip](https://github.com/Blinue/Magpie/files/7072215/Magpie.zip)
试试这个版本，然后发一下报错日志

**Comment by nt-cubic** [comment]


**Reply by Blinue** [reply]
日志里的错误很清楚。要为json文件指定一个默认程序。

**Reply by nt-cubic** [reply]
了解，非常感谢

**Comment by Tubunu** [comment]
0.6里面FFX的FsrRcas sharpness就是0.52的FSR sharpness吗

**Reply by Blinue** [reply]
它们作用相同，但FsrRcas的sharpness可以大于1

**Comment by urielejh** [comment]
Hi. this app is awesome as it brings a tool (fsr) which is very needed in certain situations. As of now i'm testing Control. i set the resolution to 3440*1440 windowed (my monitor's) and the renderer to 2646x1108 but.. how do i set the scaling? i'd like to use the ultra quality preset. also, ain't sharpness a lil bit too much? and.. i also tested lossless scaling and it feels quite much smoother.. don't know if it is the injection type or something else.
Anyway, Magpie is open-source (and free) and i'm here to test it out.

**Reply by neoKushan** [reply]
@urielejh I could be wrong, but my understanding is that the "Quality presets" are just to determine the internal -> output resolution. You've already done that. If you want better quality, change the resolution the game runs at.

**Reply by urielejh** [reply]
are you saying that when i set the resolution, Magpie runs accordingly? eg. i set 2646x1108 and Magpie knows that it has to be scaled up to 3440x1440, hence "ultra quality"?

thanks. is there a way to reduce the sharpening? do i have to change the 0.87, in the config file, to something else, like 0.70?

also, what is the "intermediate texture precision"?

**Reply by Blinue** [reply]
> is there a way to reduce the sharpening? do i have to change the 0.87, in the config file, to something else, like 0.70?

Yes, this value determines the sharpness of FSR.

> what is the "intermediate texture precision"?

This option is for debugging and you should not change it.

**Reply by urielejh** [reply]
thank you @Blinue 

i'm looking at this https://atyuwen.github.io/posts/optimizing-fsr/ which has been implemented by "lossless scaling" - altho' for its 'lite' version, i think - tool. can it be useful?

**Comment by fungnoth** [comment]
Steam上面的Lossless Scaling
你覺得是不是直接抄了你的code
